package.json

```
@@ -19,8 +19,8 @@
     "start": "node src/index.js",
     "coverage": "nyc report --reporter=cobertura --reporter=lcov"
   },
-  "dependencies": {
-    "node-fetch": "^2.6.1"
+  "engines": {
+    "node": "^18.14"
```

Isn't this going to at least raise a warning with node19+ ? How about:

```suggestion
    "node": ">=18.14"
```
?

---

right, updated